landavailability/webclient/views.py

```diff
@@ -18,7 +18,7 @@ class SearchView(TemplateView):
     def get(self, request, *args, **kwargs):
         context = self.get_context_data(**kwargs)
 
-        postcode = request.GET.get('location')
+        postcode = request.GET.get('location') or request.GET.get('postcode')
```

Comment:
The current ui passes location as a convenience since we have one input box for both postcode and place name. However plenty of tests use postcode and it seems reasonable to support both.